onnxruntime/python/tools/transformers/shape_infer_helper.py

```diff
@@ -0,0 +1,115 @@
+import os
+import sys
+
+# In ORT Package the symbolic_shape_infer.py is in ../tools
+file_path = os.path.dirname(__file__)
+if os.path.exists(os.path.join(file_path, "../tools/symbolic_shape_infer.py")):
+    sys.path.append(os.path.join(file_path, '../tools'))
+else:
+    sys.path.append(os.path.join(file_path, '..'))
+from symbolic_shape_infer import *
```

Comment:
maybe something like:
```
try:
    from onnxruntime.tools.symbolic_shape_infer import *
except ImportError:
    from ..symbolic_shape_infer import *

Reply:
will consider this later since contrib op will add to symbolic_shape_infer.py this time